src/main/java/org/folio/service/SuffixService.java

```diff
@@ -75,4 +75,20 @@ private CompletableFuture<Void> checkSuffixNotUsed(Suffix suffix, RequestContext
         }
       });
   }
+
+  public CompletableFuture<Void> isSuffixAvailable(String suffixName, RequestContext requestContext) {
```

Comment:
I disagree with the naming of this class. It does not imply that prefix can be used with purchase orders. I believe the methods in this class should be a part of a service that manages purchase orders.  It is made worse now that it is needed by other services' construction.

Reply:
I suggest to rename method name like this : "validateSuffixAvailability"

Reply:
@Abdulkhakimov  Fixed